.github/containers/trafficserver-alpine/docker-compose.yml

```diff
@@ -23,5 +23,22 @@ services:
       dockerfile: Dockerfile
       args:
         ATS_VERSION: ${ATS_VERSION}
+      # This docker-compose file is no longer used by the container-trafficserver-alpine.yml
+      # Github Actions workflow and is here
```

Comment:
> ... and is here

Seems like the sentence is missing its end?

Reply:
Ended sentence in 42b3f57f63